Common/core/platform.h

```diff
@@ -102,6 +102,10 @@
     #define AGS_PLATFORM_DEBUG  (0)
 #endif
 
+#if defined(AGS_PLATFORM_DEBUG)
+    #define AGS_PLATFORM_TEST   (1)
+#endif
+
```

Comment:
These three lines don't make much sense since AGS_PLATFORM_DEBUG is always defined. Perhaps you meant the code below?
```
#ifndef AGS_PLATFORM_TEST
#define AGS_PLATFORM_TEST (AGS_PLATFORM_DEBUG)
#endif
```

Reply:
No, I meant rather 

```
#if AGS_PLATFORM_DEBUG
#define AGS_PLATFORM_TEST (1)
#endif
```

Reply:
Can the define be also in the vcxproj (in some of the million configs in it)? So it can be set in a release build of the tests.

Reply:
I added this macro to both configs in the test projects, as intended.